qa/sc_mempool_min_fee_rate.py

```diff
@@ -0,0 +1,139 @@
+        )
+        sc_node2_configuration = SCNodeConfiguration(
+            MCConnectionInfo(address="ws://{0}:{1}".format(mc_node.hostname, websocket_port_by_mc_node_index(0))),
+            mempool_min_fee_rate=self.mempool_min_fee_rate
+        )
+        network = SCNetworkConfiguration(
+            SCCreationInfo(mc_node, forward_amount=100, withdrawal_epoch_length=self.sc_withdrawal_epoch_length),
+            sc_node_configuration,
+            sc_node2_configuration)
+        self.sc_nodes_bootstrap_info = bootstrap_sidechain_nodes(self.options, network)
+
+    def sc_setup_nodes(self):
+        return start_sc_nodes(self.number_of_sidechain_nodes, self.options.tmpdir)
+
+    def run_test(self):
+        time.sleep(0.1)
+        self.sync_all()
+        sc_node1 = self.sc_nodes[0]
+        sc_node2 = self.sc_nodes[1]
+        connect_sc_nodes(sc_node1, 1)  # Connect SC nodes
+        mc_node1 = self.nodes[0]
+
+        sc_address_1 = http_wallet_createPrivateKey25519(sc_node1)
+        sc_address_2 = http_wallet_createPrivateKey25519(sc_node2)
+
+        # Generate 1 SC block
+        generate_next_blocks(sc_node1, "first node", 1)
+
+        balanceBefore = http_wallet_balance(sc_node1)
+
+        # We need regular coins (the genesis account balance is locked into forging stake), so we perform a
+        # Forward transfer to sidechain
+        forward_transfer_to_sidechain(self.sc_nodes_bootstrap_info.sidechain_id,
+                                      mc_node1,
+                                      sc_address_1,
+                                      self.sc_nodes_bootstrap_info.genesis_account_balance,
+                                      mc_node1.getnewaddress())
+        self.sc_sync_all()
+        generate_next_blocks(sc_node1, "first node", 1)
+        self.sc_sync_all()
+
+        balanceAfter = http_wallet_balance(sc_node1)
+
+        #check forward transfer arrived correctly
+        assert_true((balanceAfter-balanceBefore) == (self.sc_nodes_bootstrap_info.genesis_account_balance * 100000000))
+
+        #generate utxo on node1
+        error = False
+        try:
+            sendCointsToMultipleAddress(sc_node1, [sc_address_1]*2, [10000]*2, 200)
+        except:
+            error = True
+        assert_false(error)
+
+        self.sc_sync_all()
+        generate_next_blocks(sc_node1, "first node", 1)
+        self.sc_sync_all()
+
+        #add tx with low feerate
+        error = False
+        try:
+            txLowId = sendCoinsToAddress(sc_node1, sc_address_2, 1000, 80)
+            #this tx has size 174 and feerate 459
+        except:
+            error = True
+        assert_false(error)
+
+        #add tx with higher feerate
+        error = False
+        try:
+            txHighId = sendCoinsToAddress(sc_node1, sc_address_2, 1000, 200)
+            #this tx has size 174 and feerate 1149
+        except:
+            error = True
+        assert_false(error)
+
+        time.sleep(5)
```

Comment:
I guess you are not using sync_mempool() because the nodes mempool will never be the same. Please add a comment.